dev_scripts/find_images_to_rebuild.py

```diff
@@ -50,5 +62,21 @@
     dry_run=True, **kwargs)
 
 handler = RebuildImagesOnRPMAdvisoryChange()
+
+if args.cassete_path:
+    VCRPY_MODE = "none"
```

Comment:
You need to set the Freshmaker configuration `vcrpy_mode` to `"none"`. I don't see you doing this in the code.